jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/PropertyRule.java

```diff
@@ -129,6 +129,24 @@ private boolean isRequired(String nodeName, JsonNode node, Schema schema) {
         return false;
     }
 
+    private boolean usesOptional(String nodeName, JsonNode node, Schema schema) {
+        if (node.has("java_optional")) {
```

Comment:
Could you rename these to `javaOptional`, as this is the naming convention for our extension properties.